src/Tools/RapidResponseDesk/sections/AreaTime/AreaAndTimeSection.jsx

```diff
@@ -1,18 +1,321 @@
+
+    setTimespanArray((prevState) => prevState.filter((item, index) => index !== id));
+  };
+
+  const setTimespanPicker = (timespan, index) => (
+    <TimespanPicker
+      id="aoi-time-select"
+      minDate={minDateRange}
+      maxDate={maxDateRange}
+      datePickerInputStyle={{ width: '85px' }}
+      timespan={{ fromTime: timespan.from, toTime: timespan.to }}
+      applyTimespan={updateTimespan(index)}
+      timespanExpanded={true}
+      calendarHolder={cardHolderRef}
+      displayCalendarFrom={timespan.displayCalendarFrom}
+      openCalendarFrom={() => updateCalendarOpenState(timespan, true)}
+      closeCalendarFrom={() => updateCalendarOpenState(timespan, false)}
+      displayCalendarUntil={timespan.displayCalendarTo}
+      openCalendarUntil={() => updateCalendarOpenState(timespan, true, false)}
+      closeCalendarUntil={() => updateCalendarOpenState(timespan, false, false)}
+      showNextPrevDateArrows={true}
+      getAndSetNextPrevDateFrom={async (direction, selectedDay) =>
+        await getAndSetNextPrevDate(direction, selectedDay, index, timespan)
+      }
+      getAndSetNextPrevDateTo={async (direction, selectedDay) =>
+        await getAndSetNextPrevDate(direction, selectedDay, index, timespan, false)
+      }
+      isDisabled={false}
+    />
+  );
+
+  const setDateRangeContainer = () => (
+    <div className="date-picker-container">
+      {timespanArray.map((timespanFrame, index) => {
+        return (
+          <div
+            key={`date-picker-${index}`}
+            className={`date-picker-content${index + 1 < timespanArray.length ? ' bottom-border' : ''}`}
+          >
+            {setTimespanPicker(timespanFrame, index)}
+            {timespanArray.length > 1 && (
+              <div className="remove-button-container">
+                <Button
+                  icon={'fas fa-trash'}
+                  type={ButtonType.text}
+                  rounded={true}
+                  onClick={() => removeTimespanFromList(index)}
+                ></Button>
+              </div>
+            )}
+            {timespanArray.length === index + 1 && (
+              <div className="add-button-container">
+                <Button
+                  icon={'fas fa-plus'}
+                  type={ButtonType.text}
+                  iconStyle={{ transform: 'scale(0.85)' }}
+                  rounded={true}
+                  onClick={() => setAdditionalDateRange(timespanFrame)}
+                ></Button>
+              </div>
+            )}
+          </div>
+        );
+      })}
+      <div className="calendar-holder" ref={cardHolderRef} />
+    </div>
+  );
+
+  const setBody = () => (
+    <div className="area-time-body">
+      <div className="area-interest-container">
+        <AOISelection
+          aoiGeometry={aoiGeometry}
+          aoiIsDrawing={aoiIsDrawing}
+          mapBounds={mapBounds}
+        ></AOISelection>
+        <div className="coverage-slider-container">
```

Comment:
I need to clarify this confusion with Daniel and Kristijan, since in the [issue](https://docs.google.com/document/d/1MU6X7cSZqCVaTcU3JyCo9cKm5JHb5Ps1sngAHGQHBi0/edit#heading=h.kq34havt4q5u) I can see that `AOI coverage slider` should be present but in the designs there is no sight of it

Reply:
I see that the gdoc file was not updated properly (sorry for that), with some feedback that we got from Grega/ESA, the AOI coverage slider was moved to the `Advanced` tab so please remove all the logic related to it and we will re-add it back later